decision-policy.php

```diff
@@ -14,27 +14,37 @@
 		<ol>
 			<li> Discussion on a topic proceeds until the chairs believe that all points of view have been expressed and the group has considered the variety of information presented. Depending on the topic, this discussion may take a couple of days or a couple of weeks, or more. 
 				<ol>
-					<li> Discussion can take place in any <a href="./communication">recognized channel of the Working Group</a> including email on the AG mailing list, comment threads for GitHub issues or pull requests, or on Working Group calls. </li>
+					<li> Discussion can take place in any <a href="./communication">recognized channel of the Working Group</a> including email on the public AG mailing list, AG surveys, comment threads for GitHub issues or pull requests, or on Working Group calls. </li>
 				</ol>
 			</li>
 			<li> When the chairs believe that the group is ready to come to a decision they announce a Call for Consensus by email to the Working Group's mailing list. The Call must remain open for a minimum of two working days. 
 				<ol>
-					<li> The Call is open to responses from all group members. </li>
+					<li> The Call is open to responses from all <a href="https://www.w3.org/groups/wg/ag/participants">group participants</a>. </li>
 					<li>The Call will be for a single topic, will clearly indicate that it is a Call for Consensus, and will contain pointers to the relevant discussion. This may include links to GitHub issues or pull requests, AG surveys, email threads, or meeting minutes.</li>
 					<li> A resolution recorded in a WG teleconference may precede a Call for Consensus, but it may not replace the official Call for Consensus. </li>
 					<li> Issues that are regarded as editorial by the Chairs do not require a Working Group decision in order for the Chairs to address, and thus do not require a Call for Consensus. If there is disagreement by participants on whether something is editorial this can be brought to the attention of the chairs either privately or in the context of the wider group. </li>
 				</ol>
 			</li>
+			<li>Responding to the Call for Consensus<ol>
+					<li>Individuals are invited, but not required, to support the decision by replying to the CfC email with “+1”, or to indicate a “can live with” status by replying with a “+0”.</li>
+					<li>Objections can be raised by replying to the CfC email with “-1”. In order to facilitate understanding and discussion, it is recommended to include a rationale which includes:<ol>
+							<li>why the individual objects to the proposed decision;</li>
+							<li>describes how the objection was raised and considered during discussion leading to the CfC, including how reactions to the potential objection were addressed;</li>
+							<li>what alternate decision would remove the objection <b>and</b> be likely to gain consensus in the WG.</li>
+						</ol></li>
+					<li>Objections are subject to examination by the WG and therefore must be filed on the public record.</li>
```

Comment:
Does this include who raised the objection or just that the content of the objection needs to be public?

Reply:
@awkawk if they are raising objection via an email response, doesn't that automatically provide a 'who'

Reply:
Not if they email the chairs only to raise the objection.